godtools/App/Features/AppLanguage/Domain/UseCases/GetConfirmAppLanguageInterfaceStringsUseCase.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  GetConfirmAppLanguageInterfaceStringsUseCase.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 10/23/23.
+//  Copyright © 2023 Cru. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+class GetConfirmAppLanguageInterfaceStringsUseCase {
+    
+    private let getCurrentAppLanguageUseCase: GetCurrentAppLanguageUseCase
+    private let getInterfaceStringInAppLanguageUseCase: GetInterfaceStringInAppLanguageUseCase
+    private let getInterfaceStringRepositoryInterface: GetInterfaceStringForLanguageRepositoryInterface
+    private let localeLanguageName: LocaleLanguageName
+    
+    init(getCurrentAppLanguageUseCase: GetCurrentAppLanguageUseCase, getInterfaceStringInAppLanguageUseCase: GetInterfaceStringInAppLanguageUseCase, getInterfaceStringRepositoryInterface: GetInterfaceStringForLanguageRepositoryInterface, localeLanguageName: LocaleLanguageName) {
+        
+        self.getCurrentAppLanguageUseCase = getCurrentAppLanguageUseCase
+        self.getInterfaceStringInAppLanguageUseCase = getInterfaceStringInAppLanguageUseCase
+        self.getInterfaceStringRepositoryInterface = getInterfaceStringRepositoryInterface
+        self.localeLanguageName = localeLanguageName
+    }
+    
+    func getStringsPublisher(for selectedLanguage: AppLanguageCodeDomainModel) -> AnyPublisher<ConfirmAppLanguageInterfaceStringsDomainModel, Never> {
+        
+        return Publishers.CombineLatest4(
```

Comment:
Hey @rachaelblue I'm also thinking about deprecating/removing the use case ```GetInterfaceStringInAppLanguageUseCase```.  

I realized that this use case can get very cumbersome to use when we have a lot of interface strings to translate and it also will create more overhead to have a publisher for each interface string.  For example the ToolDetails has about 10 interface strings to stuff into a domain model. 

Instead we can ask the data layer interface to translate our interface strings and the input would be the language to translate strings into.  For example in Tool Details it needs interface strings translated into the tool language.

https://github.com/CruGlobal/godtools-swift/blob/develop/godtools/App/Features/ToolDetails/Data-DomainInterface/GetToolDetailsInterfaceStringsRepository.swift

I'm going to create a separate ticket for this too so we can get this PR merged in.